exercises/transpose/transpose.spec.js

```diff
@@ -0,0 +1,65 @@
+var transpose = require('./transpose');
+
+describe('Transpose', function () {
+  it('test empty string', function () {
+    expect(transpose('')).toEqual('');
+  });
+
+  xit('test two characters in a row', function () {
+    var input = 'A1';
+    var expected = 'A\n1';
+    expect(transpose(input)).toEqual(expected);
+  });
+
+  xit('test two characters in a column', function () {
+    var input = 'A\n1';
```

Comment:
When I look at the `problem-specifications` repo, it may be slightly unclear what the expected form of the result is.  However, the examples all take arrays as input and expect arrays as results.

I think it would be best if you altered your solution to match the API described in the problem-spec repo, rather than have one that consumes and returns strings.

Reply:
Yep, I agree with @matthewmorgan, `transpose` should accept an `array` as parameter and return an `array` as a result.

@kk-r, you'll find the canonical data @matthewmorgan is talking about here: https://github.com/exercism/problem-specifications/blob/master/exercises/transpose/canonical-data.json